packages/components/ng-ovh-feature-flipping/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@ovh-ux/ng-ovh-feature-flipping",
+  "version": "1.0.0",
+  "description": "Feature flipping library",
+  "keywords": [
+    "angularjs",
+    "feature-flipping",
+    "ovh",
+    "ovhcloud"
+  ],
+  "homepage": "https://github.com/ovh/manager/tree/master/packages/components/ng-ovh-feature-flipping#readme",
+  "bugs": {
+    "url": "https://github.com/ovh/manager/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/components/ng-ovh-feature-flipping"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/umd/ng-ovh-feature-flipping.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup -c --environment BUILD:production",
+    "dev": "rollup -c --environment BUILD:development",
+    "dev:watch": "yarn run dev --watch",
+    "prepare": "yarn run build",
+    "start": "lerna exec --stream --scope='@ovh-ux/ng-ovh-feature-flipping' --include-dependencies -- yarn run build",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/ng-ovh-feature-flipping' --include-dependencies -- yarn run dev",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/ng-ovh-feature-flipping' --include-dependencies -- yarn run dev:watch"
+  },
+  "devDependencies": {
+    "@ovh-ux/component-rollup-config": "^7.0.0"
+  },
+  "peerDependencies": {
+    "angular": "~1.6"
```

```suggestion
    "angular": "^1.7"
```
according to version used in manager